worker/src/main/java/gov/cms/ab2d/worker/properties/PropertiesChangeDetection.java

```diff
@@ -0,0 +1,19 @@
+package gov.cms.ab2d.worker.properties;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.scheduling.annotation.Scheduled;
+import org.springframework.stereotype.Component;
+
+
+@Component
+class PropertiesChangeDetection {
+
+    @Autowired
+    private PropertiesInit propertiesInit;
+
+    // Every 10 minutes
+    @Scheduled(fixedDelay = 10 * 60 * 1000)
```

Comment:
We definitely want property changes to take effect sooner. Let's do 20 seconds instead.